src/SimplePDPService.sol

```diff
@@ -112,9 +115,10 @@ contract SimplePDPService is PDPListener, PDPRecordKeeper, Initializable, UUPSUp
     }
 
     // The start of the challenge window for the current proving period
+    // If proof set state not yet initialized assume proving period starts now
     function thisChallengeWindowStart(uint256 setId) public view returns (uint256) {
-        if (provingDeadlines[setId] == 0) {
-            revert("Proving not yet started");
+        if (provingDeadlines[setId] == NO_PROVING_DEADLINE) {
+            return block.number + getMaxProvingPeriod() - challengeWindow();
```

Comment:
What's the rationale for not reverting here anymore ?

Reply:
The idea is to make this as useful as possible in the error case.  In curio code I keep getting thwarted by this error when what I really want is for it to return to me the value I should call to nextProvingPeriod.  You could argue maybe this should belong in a second function instead and I'm still mulling that over.

Reply:
I'm going to make another function for this behavior.

Reply:
Done